devfiles/go/devfile.yaml

```diff
@@ -18,7 +18,7 @@ components:
 -
   type: dockerimage
   # this version is used in the plugin
-  image: quay.io/eclipse/che-golang-1.10:nightly
+  image: quay.io/eclipse/che-golang-1.12:nightly
```

Comment:
do we need to support the go 1.10 stuff too? or just drop it in favour of the latest?

Reply:
see also https://github.com/eclipse/che-plugin-registry/pull/224

Reply:
@nickboldt @amisevsk 
I have no idea

Reply:
I'm currently looking to deprecate 1.10 if possible, so if the devfiles work with 1.12, that makes my life easier.

See: https://github.com/eclipse/che/issues/14265, https://github.com/eclipse/che-devfile-registry/pull/79#discussion_r313518538